src/net.h

```diff
@@ -424,6 +428,7 @@ class CNode
             case ConnectionType::MANUAL:
             case ConnectionType::ADDR_FETCH:
             case ConnectionType::FEELER:
+            case ConnectionType::MANUAL_BLOCK_RELAY:
                 return false;
```

Comment:
It's used in `net_processing` in places suchas  `ConsiderEviction()` where we wouldn't want to consider manual connections. 
I think the name "IsOutboundOrBlockRelayConn" isn't ideal (even before this PR, because manual connections are  also outbound connections), maybe the name "IsAutomaticOutboundOrBlockRelayConn" would be better. I added a commit to do that.